src/api/decoding-scanned-codes.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+parent: API Reference
+title: Decoding Scanned Codes
+permalink: /api/decoding-scanned-codes
+---
+
+# Decoding Scanned Codes
+
+A scanned code can be a either QR code or a barcode that a merchant or a patron scans.
+The scanned code can be decoded to provide information about the code.
+
+## Contents
+{:.no_toc .text-delta}
+
+* TOC
+{:toc}
+
+## Models
+
+### Scanned Code
+
+{% h4 Mandatory Fields %}
+
+|    Field    |  Type  |                           Description                           |
+| :---------- | :----- | :-------------------------------------------------------------- |
+| code        | String | The utf8 representation of data decoded from what was scanned.  |
+| scannedBy   | String | The person who scanned the code. Can be `merchant` or `patron`. |
+| provider    | String | The party that provided the code.                               |
```

Comment:
"The party that provided the code" is not clear, the person scanning provided a code too. Your example has "Farmlands", do you have words that could describe what this means?

We will have specific integrators looking to this to figure out how to provide line items for Payment further along. Perhaps you could refer to that?

Reply:
I've change the description of this field and added some context around line items too.